lib/node_modules/@stdlib/math/base/special/sech/test/test.native.js

```diff
@@ -0,0 +1,198 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require('tape');
+var isnan = require('@stdlib/math/base/assert/is-nan');
+var isPositiveZero = require('@stdlib/math/base/assert/is-positive-zero');
+var PINF = require('@stdlib/constants/float64/pinf');
+var NINF = require('@stdlib/constants/float64/ninf');
+var EPS = require('@stdlib/constants/float64/eps');
+var abs = require('@stdlib/math/base/special/abs');
+var sech = require('./../lib/native.js');
```

Comment:
You cannot assume that you can load `./lib/native.js`. See other `math/base/special/*` packages with C implementations and emulate how they handle native add-on tests.